core/arch/arm/plat-rcar/romapi_call.S

```diff
@@ -31,9 +32,11 @@ FUNC __plat_romapi_wrapper , : , .identity_map
 	/*
 	 * We are running at identity location, so we can't use bl there,
 	 * because assembler will generate relative address to virt_to_phys(),
-	 * which is not identity mapped. So we need to use absolute address.
+	 * which is not identity mapped.
 	 */
-	ldr	x9, =virt_to_phys
+	adr     x9, virt_to_phys
```

Comment:
This should probably be an `adr_l` instead since we don't know how far away it might be, same below.

Reply:
Yes, will fix.